File: HRM.UI/src/app/components/HRM/Management/NhanVienManagement/Model/nhan-vien.model.ts
export interface NhanVienDTO {
  id_NV: number;
  maNV?: string;
  hoTen?: string;
  ngaySinh?: string;
  cmnd?: string;
  mobile?: string;
  email?: string;
  sotaikhoan?: string;
}

export interface NhanVienModel {
  maNV?: string;
  holot?: string;
  ten?: string;
  ngaySinh?: string;
  cmnd?: string;
  mobile?: string;
  email?: string;
  sotaikhoan?: string;
}

export interface TableState {
  paginator: {
    pageIndex: number;
    pageSize: number;
    total: number;
  };
  filter: any;
  sorting: {
    column: string;
    direction: string;
  };
  searchTerm: string;
}

File: HRM/HRM/Model/NhanVien/NhanVienModel.cs
using System.ComponentModel.DataAnnotations;

namespace HRM.Model.NhanVien
{
    public class NhanVienModel
    {
        [Required]
        public string MaNV { get; set; } = null!;

        public string? Holot { get; set; }

        [Required]
        public string Ten { get; set; } = null!;

        public DateTime? Ngaysinh { get; set; }
        public string? CMND { get; set; }
        public string? Mobile { get; set; }

        [EmailAddress]
        public string? Email { get; set; }
    }
}
